src/renderer/components/menu/menu.tsx

```diff
@@ -35,11 +36,16 @@ export interface MenuProps {
 }
 
 interface State {
-  position?: MenuPosition;
+  position?: {
+    left?: boolean;
+    top?: boolean;
+    right?: boolean;
+    bottom?: boolean;
+  };
```

Comment:
Why are these booleans? Furthermore, it doesn't look like this is modifiable so it isn't really state.

Reply:
It's different type of positioning (currently auto-positioning only) when `usePortal={true}`